WordPress/Classes/ViewRelated/Blog/Blog Dashboard/BlogDashboardViewController.swift

```diff
@@ -190,12 +190,8 @@ final class BlogDashboardViewController: UIViewController {
         viewModel.loadCards()
     }
 
-    /// Load card from cache if view is appearing
+    /// Load card from cache
     @objc private func loadCardsFromCache() {
-        guard view.superview != nil else {
-            return
-        }
-
```

Comment:
@leandroalonso I've removed this check to fix a minor edge case bug.

The bug:

1. Set Initial Screen to be Home
2. Enable quick start for a site
3. Navigate to Menu
4. Change Initial Screen to be Menu
5. Dismiss settings
6. Navigate to Home
7. Notice that the QS is displayed for a second before it gets removed

This was fixed by removing this check. I thought it was safe to remove this check since reloading locally isn't a high-performance task. Do you agree, or should we keep it?